### PR TITLE
Adding support to decide whether final results must be encrypted

### PR DESCRIPTION
**Pull request by hugoseabra-imachines**
- Added new crypto module with better encrypting definitions, façade and tests dedicated to it;
- Added dedicated folder to run tests;
- Added option to support decision whether data being uploaded to storage must be encrypted or plain;
- Added flag to decide whether final results in Job Bulk payouts must be encrypted or plain;

### CODE REVIEW

File: test/Escrow.test.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ contract('Escrow', (accounts) => {
 
     it('fails when aborting with different address', async () => {
       try {
-        tx = await Escrow.abort({ from: externalAddress });
```
Comment (igervasiychuk) [inline test/Escrow.test.js:83]: this doesn't look correct, tx is used on the next line. 
and I think this test won't break if someone were to remove the "trusted" modifier on the "abort" method in the escrow contract

File: bin/wait_then_run [hunk 2]
```diff
@@ -30,7 +30,5 @@ export PYTHONPATH=$(pwd)
 if [ ! -z $* ]; then
   $*
 else
-  python3 hmt_escrow/job.py
-  python3 hmt_escrow/storage.py
-  python3 hmt_escrow/eth_bridge.py
+  python3 -m unittest discover -s test/hmt_escrow
```
Comment (igervasiychuk) [inline bin/wait_then_run:31]: I would've created two PRs, one with the refactoring, and one with the two-liner for the "Adding support to decide whether final results must be encrypted".

**Review by igervasiychuk — CHANGES_REQUESTED** [review]

File: bin/wait_then_run (hunk 2 above)
Comment (hugoseabra-imachines) [reply bin/wait_then_run:31]: Great point. I should've done that, indeed.

Would you like me to separate the both implementations in different PR?

Comment (hugoseabra-imachines) [reply bin/wait_then_run:31]: @igervasiychuk , I will close this PR in favor of PR split to other 3:
- https://github.com/humanprotocol/hmt-escrow/pull/328
- https://github.com/humanprotocol/hmt-escrow/pull/329

**Pull request closed without merging**